src/Data/String.purs

```diff
@@ -72,7 +73,7 @@ foreign import _toChar :: (forall a. a -> Maybe a)
 
 -- | Returns `true` if the given string is empty.
 null :: String -> Boolean
-null s = length s == zero
+null = eq mempty
```

Comment:
I knew there must've been a reason why `mempty` isn't being used here. Is a JSON lookup amortized O(1)?